portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase

```diff
@@ -1648,7 +1648,7 @@ definition {
 			totalItemsCount = "1");
 
 		GDPR.editEntryCP(
-			applications = "Documents and Media",
+			dmDocumentTitle = "DM Document Title",
```

Comment:
These two commits can be combined into one, thanks

Reply:
Done